setup.py

```diff
@@ -41,6 +41,7 @@ def run_tests(self):
     },
     tests_require=['tox',
                    'pytest',
-                   'pytest-cov'],
+                   'pytest-cov',
```

Comment:
Because pytest needs to be installed in the virtualenv, and this won't do that. I suspect the tests only work currently because you are installing requirements.txt in the Makefile, and you are having tox invoke make.